metadata-ingestion/source_docs/kafka.md

```diff
@@ -69,6 +69,23 @@ sink:
   # sink configs
 ```
 
+If you are trying to add domains to your topics you can use a configuration like below.
+
+```yml
+source:
+  type: "kafka"
+  config:
+    # ...connection block
+    domain:
+      "urn:li:domain:13ae4d85-d955-49fc-8474-9004c663a810":
+        allow:
+          - ".*"
+      "urn:li:domain:d6ec9868-6736-4b1f-8aa6-fee4c5948f17":
+        deny:
+          - ".*"
+```
+
+Note that the `domain` in config above can be either an _urn_ or a domain _id_ (i.e. `urn:li:domain:13ae4d85-d955-49fc-8474-9004c663a810` or simply `13ae4d85-d955-49fc-8474-9004c663a810`). The Domain should exist in your DataHub instance before ingesting data into the Domain. To create a Domain on DataHub, check out the [Domains User Guide](https://datahubproject.io/docs/domains/).
```

Comment:
Small comment- domains can also be set for other sources as well. We might want to add this disclaimer to the others also. Or centralize it!